app/helpers/template_helpers.py

```diff
@@ -214,34 +230,46 @@ def get_census_base_url(schema_theme: str, language_code: str) -> str:
 
 @lru_cache(maxsize=None)
 def get_static_content_urls(language_code: str, base_url: str, schema_theme: str):
+    is_nisra_theme = schema_theme == "census-nisra"
     if language_code == "cy":
         help_path = "help/sut-i-ateb-y-cwestiynau/help-y-cwestiynau-ar-lein/"
         cookies_path = "cwcis/"
-        accessibility_statement_path = "hygyrchedd/"
+        accessibility_statement_path = "datganiad-hygyrchedd/"
         privacy_and_data_protection_path = "preifatrwydd-a-diogelu-data/"
         terms_and_conditions_path = "telerau-ac-amodau/"
         contact_us = "cysylltu-a-ni/"
-
+        languages = "help/ieithoedd-a-hygyrchedd/ieithoedd/"
+        bsl_and_audio_videos = (
+            "help/ieithoedd-a-hygyrchedd/hygyrchedd/fideos-hygyrch-gyda-bsl/"
+        )
     else:
         help_path = (
             "help/help-with-the-questions/online-questions-help/"
-            if schema_theme == "census-nisra"
+            if is_nisra_theme
             else "help/how-to-answer-questions/online-questions-help/"
         )
         cookies_path = "cookies/"
-        accessibility_statement_path = "accessibility/"
+        accessibility_statement_path = "accessibility-statement/"
         privacy_and_data_protection_path = "privacy-and-data-protection/"
         terms_and_conditions_path = "terms-and-conditions/"
         contact_us = "contact-us/"
-
-    return {
+        languages = "help/languages-and-accessibility/languages/"
+        bsl_and_audio_videos = (
+            "help/languages-and-accessibility/accessibility/accessible-videos-with-bsl/"
+        )
+    item_list = {
         "help": f"{base_url}{help_path}",
         "cookies": f"{base_url}{cookies_path}",
         "accessibility_statement": f"{base_url}{accessibility_statement_path}",
         "privacy_and_data_protection": f"{base_url}{privacy_and_data_protection_path}",
         "terms_and_conditions": f"{base_url}{terms_and_conditions_path}",
         "contact_us": f"{base_url}{contact_us}",
     }
+    if not is_nisra_theme:
+        item_list["languages"] = f"{base_url}{languages}"
+        item_list["bsl_and_audio_videos"] = f"{base_url}{bsl_and_audio_videos}"
+
+    return item_list
```

Comment:
`item_list` is a bit generic, `urls` would be better:
```suggestion
    urls = {
        "help": f"{base_url}{help_path}",
        "cookies": f"{base_url}{cookies_path}",
        "accessibility_statement": f"{base_url}{accessibility_statement_path}",
        "privacy_and_data_protection": f"{base_url}{privacy_and_data_protection_path}",
        "terms_and_conditions": f"{base_url}{terms_and_conditions_path}",
        "contact_us": f"{base_url}{contact_us}",
    }
    if not is_nisra_theme:
        urls["languages"] = f"{base_url}{languages}"
        urls["bsl_and_audio_videos"] = f"{base_url}{bsl_and_audio_videos}"

    return urls
```